src/gui/shareusergroupwidget.cpp

```diff
@@ -522,9 +521,9 @@ ShareUserLine::ShareUserLine(AccountPtr account,
     showNoteOptions(false);
 
     // email shares do not support notes and expiration dates
```

Comment:
please update the comment or better remove it
the code is obvious not sure I need a comment to help reading it